lib/themes/dosomething/paraneue_dosomething/js/utilities/Analytics.js

```diff
@@ -44,14 +44,19 @@ function subscribeEvents() {
   // Signup button on pitch page
   const isPitchPage = $('.pitch').length > 0;
   if (isPitchPage) {
+    $('#link--campaign-signup-login').on('click', () => {
```

Comment:
To make the math easier for Luke, right now it's incredibly difficult / impossible for him to calculate a specific number.